libcxx/src/experimental/time_zone.cpp

```diff
@@ -29,6 +29,14 @@
 // These quirks often use a 12h interval; this is the scan interval of zdump,
 // which implies there are no sys_info objects with a duration of less than 12h.
 
+// Work around https://gcc.gnu.org/bugzilla/show_bug.cgi?id=120502
```

Comment:
Maybe add a `TODO(GCC)`?

Reply:
Added a `TODO(LLVM 23)`, since we know that we can drop this when we upgrade GCC next time.